### PR TITLE
chore: create test scaffolding in server-ct

### PR DESCRIPTION
**Pull request by elevatebart**
Set up a way to run tests in Typescript using mocha in server-ct.

### CODE REVIEW

File: packages/server-ct/index.js [hunk 1]
```diff
@@ -1,5 +1,5 @@
 const browsers = require('@packages/server/lib/browsers')
-const Project = require('./src/project-ct').default
+const Project = require('./dist/project-ct').default
```
Comment (elevatebart) [inline packages/server-ct/index.js:2]: should not be done in this PR

**Review by elevatebart — COMMENTED** [review]
I have one change to make before it is ready to review

File: packages/server-ct/package.json [hunk 2]
```diff
@@ -2,7 +2,12 @@
...
   "private": true,
-  "files": [
-    "src"
-  ]
+  "main": "index.js",
+  "scripts": {
+    "test-unit": "ts-mocha --config ./test/.mocharc.js"
+  },
+  "devDependencies": {
+    "@types/mocha": "5.2.7",
```
Comment (lmiller1990) [inline packages/server-ct/package.json:10]: seems they have more up-to-date types: https://www.npmjs.com/package/@types/mocha

File: packages/server-ct/src/project-ct.ts [hunk 3]
```diff
@@ -304,7 +303,7 @@ export default class Project {
         this.emit('socket:connected', id)
       },
 
-      onSetRunnables (runnables: Runnables) {
+      onSetRunnables (runnables: RootRunnable) {
```
Comment (lmiller1990) [inline packages/server-ct/src/project-ct.ts:306]: thanks, how did you figure out these types?

**Review by lmiller1990 — APPROVED** [review]
LGTM, but I recommend updating the @types/mocha

File: packages/server-ct/src/project-ct.ts (hunk 3 above)
Comment (elevatebart) [reply packages/server-ct/src/project-ct.ts:306]: When I added a tsconfig.json, the file started to be all red...
I followed the path of the F12.

File: packages/server-ct/package.json (hunk 2 above)
Comment (elevatebart) [reply packages/server-ct/package.json]: And I am not even using the ones I installed

**Pull request merged**